projects/plugins/jetpack/readme.txt

```diff
@@ -242,27 +242,71 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 10.8-a.9 - 2022-03-23
+### 10.8-beta - 2022-03-29
 #### Major Enhancements
 - QR Post: add new feature which automatically generates QR codes for published posts. When scanned, the QR code will link visitors to the post. If a site has a custom logo set, it will be shown in the generated QR code image.
 
 #### Enhancements
+- Add back the upgrade nudge to the Payment Button block.
+- Adding language attributes to code blocks.
+- Dashboard: add a card to highlight the blocks available in Jetpack.
+- Dashboard: add highlight of the Jetpack blocks to the settings page.
+- Dashboard: add toggle to enable new Google Fonts feature.
+- Dashboard: show Search benefits on the Jetpack disconnection screen.
 - External Media: add Openverse as an external Media provider.
+- Form Block: offer option to install/activate the Jetpack CRM plugin from the Form Block options.
+- Jetpack: add QRPost feature which generates QR codes for published posts. Currently a JETPACK_BETA_BLOCKS feature.
+- Jetpack: assistant style updates and other improvements.
+- Jetpack: using the new Webfont API in Gutenberg, registers a selection of Google fonts for use with block and Global styles.
+- Jetpack Blocks: make settings discoverable and toggleable.
+- Markdown block: add default spacing controls.
+- Masterbar: make the Desktop Switcher look the same between Calypso and wp-admin.
 - Payment Block: clicking on payment links from email or WordPress.com Notifications/Reader will correctly open the corresponding payment form.
-- Dashboard: show Search benefits on the Jetpack disconnection screen.
+- Payment Blocks: add a unified introduction to payment blocks to make it easier to select the correct one.
+- Payment Blocks: Added links to manage subscriptions and fees to the Product Management controls.
+- Payment Blocks: add more keywords to payments blocks so they're easier to find.
+- Payment Button Block: make Stripe connection button visible from child block elements.
+- Payments Block: adds a link to the support reference page on the block configuration panel.
+- Payments Block: introduce new Post Publish panel to highlight the options available with Payments blocks.
+- Payments Block now uses the product management controls to manage subscriptions. Migrated the Payments Block to a functional component and cleaned up code.
```

Comment:
Now that we're squashing things, I think it makes sense to consolidate some of those things into a single entry.